Description: do not attempt to build the modules on PREEMPT_RT kernels
Author: Andreas Beckmann <anbe@debian.org>
Bug-Debian: https://bugs.debian.org/981212
Bug: https://github.com/openzfs/zfs/issues/11097
Last-Update: 2024-08-08

--- a/scripts/dkms.mkconf
+++ b/scripts/dkms.mkconf
@@ -25,6 +25,9 @@ PACKAGE_NAME="${pkgname}"
 PACKAGE_VERSION="${pkgver}"
 PACKAGE_CONFIG="${pkgcfg}"
 NO_WEAK_MODULES="yes"
+
+BUILD_EXCLUSIVE_CONFIG="!CONFIG_PREEMPT_RT"
+
 PRE_BUILD="configure
   --disable-dependency-tracking
   --prefix=/usr
